addOns/oast/src/main/javahelp/org/zaproxy/addon/oast/resources/help/contents/services/boast/options.html

```diff
@@ -17,27 +17,27 @@ <h3>Server URI</h3>
     running a working instance of BOAST.</p>
 <p>An example of a valid URI is: <code>https://example.com:1337/events</code>.</p>
 
-<h3>ID</h3>
+<h3>Polling Frequency</h3>
 
-<p>A non-editable field that is filled in when you register with a BOAST instance.</p>
-<p>This ID can be used to craft an address that can be specified in relevant attacks.</p>
-<p>For example, if an ID is <code>cxcjyaf5wahkidrp2zvhxe6ola</code>, and the Server URI host is <code>example.com</code>,
-    then the payload URI host will be <code>cxcjyaf5wahkidrp2zvhxe6ola.example.com</code>.</p>
+<p>This option allows you to change the frequency of polling the registered BOAST servers. It takes values in
+    seconds. The minimum allowed value is 10 seconds and there is no maximum allowed value. The default value is 60
```

Comment:
The minimum is not actually being enforced by the code (e.g. when called programmatically or with -config arg).